serveraddr.go

```diff
@@ -18,6 +18,9 @@ const (
 
 	// ProtocolDoH is DNS over HTTPS.
 	ProtocolDoH = Protocol("doh")
+
+	// ProtocolDoQ is DNS over QUIC.
+	ProtocolDoQ = Protocol("doq")
```

Comment:
In https://github.com/rbmk-project/dnscore/pull/19, I added aliases for protocols, which you can find right below.

Would you mind also adding an alias for `ProtocolQUIC` below?

(I added aliases here because I think the protocol names read very clearly, and I found myself typing `dnscore.ProtocolHTTPS` rather than `dnscore.ProtocolDoH` a few times, which I took as an indication that perhaps the aliases are more intuitive names than the original names, so I wanted to experiment a bit with them in tree and see whether perhaps we should consider using the aliases as official names and the official names as aliases. I am asking you to add this change for consistency with other protocols.)